### PR TITLE
Update astroquery version to 0.3.6

### PR DESCRIPTION
**Pull request by bsipocz**

### CODE REVIEW

**Comment by bsipocz** [comment]
@mwcraig - Can you see any obvious thing that goes wrong here?

**Comment by mwcraig** [comment]
@bsipocz -- two things...

+ Please make sure you fork to your personal account and open the PR from there in the future. By default, the edit web interface on github makes a branch on the repo on the branch if you own the repo, but any build from a branch is uploaded.

+ The cause of the failure is that the license file was moved in https://github.com/astropy/astroquery/commit/3c9e6e0aa5d1ea6477645d947d1474efbc2d257b but the location has not been updated in the recipe.

**Comment by bsipocz** [comment]
@mwcraig - I'm sorry about the branch. Did the update on github, and forgot that this update only works if one doesn't have commit rights :(

**Comment by mwcraig** [comment]
Yeah, I wish the web interface would let you choose where to fork. Would be really convenient...

**Comment by bsipocz** [comment]
Oh, well, indeed. The issue is that with moving the license we forgot to update MANIFEST, thus the file left out from this release. Let me know if you have a good idea to work around this issue.

**Comment by mwcraig** [comment]
I'd make a new release, even if it is just to handle the license, and build that. You might be able to drop the license_file altogether and have this one build....but you probably do want the license included in the release, so maybe better to do another release anyway.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {% set name = "astroquery" %}
-{% set version = "0.3.5" %}
-{% set sha256 = "f111de4c2281c5de3eb2972e20880fe997d287494e28d055a4802f831664d1c1" %}
+{% set version = "0.3.6.post2" %}
```
Comment (bsipocz) [inline recipe/meta.yaml]: I'm not sure how well the post release work for conda users, but we should at least try

**Pull request merged**